src/components/ManageUsers/manage-users.css

```
@@ -0,0 +1,84 @@
+.usersearch__h1 {
+  font-size: 16px;
+  color: #e3893e;
+  display: right;
+  font-weight: 200px;
+  margin-right: 10px;
+  text-align: right;
+}
+
+.usersearch__container {
+  padding: 16px;
+  min-height: 10vh;
+  background: #000000;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  margin-left: auto;
+}
+
+.usersearch__search {
+  margin-top: 10px;
+  margin-bottom: 10px;
```

it can be shortened to:
```suggestion
  margin: 10px 0;
```

---

Fixed